lib/irb/locale.rb

```diff
@@ -109,12 +109,12 @@ def require(file, priv = nil)
 
     alias toplevel_load load
 
-    def load(file, priv=nil)
+    def load(file)
```

Comment:
It looks good :+1:, but one thing, priv seems to be used in line 92.
```ruby
IRB::Locale.new.require 'irb/error.rb'
# => /Users/tomoya/Documents/github/ruby/irb/lib/irb/locale.rb:112:in `load': wrong number of arguments (given 2, expected 1) (ArgumentError)
```
I think `def require(file, priv = nil)` is not used, so it won't be a problem, but how about removing `priv=nil` from method `require`, or remove `require` method

Reply:
Yeah let's remove the `require` too.